fennel/datasets/datasets.py

```diff
@@ -229,6 +229,12 @@ def rename(self, columns: Dict[str, str]) -> _Node:
     def drop(self, columns: List[str]) -> _Node:
         return Drop(self, columns)
 
+    def select(self, columns: List[str]) -> _Node:
+        ts = self.dsschema().timestamp
+        # Keep the timestamp col
+        drop_cols = list(filter(lambda c : c not in columns and c != ts, self.dsschema().fields()))
+        return Drop(self, drop_cols)
```

Comment:
I thought we were going to pass protos back and use Drop in the backend? cc @bothra90

Reply:
We decided to make it a client side change. The code will still be visible to the user as part of the pipeline